controls/nginx_spec.rb

```diff
@@ -181,3 +194,64 @@
     its('add_header') { should include 'X-Content-Type-Options nosniff' }
   end
 end
+
+control 'nginx-11' do
+  impact 1.0
+  title 'Disable content-type sniffing'
+  desc 'It prevents browser from trying to mime-sniff the content-type of a response away from the one being declared by the server. It reduces exposure to drive-by downloads and the risks of user uploaded content that, with clever naming, could be treated as a different content-type, like an executable.'
+  describe parse_config_file(nginx_hardening, options_add_header) do
+    its('add_header') { should include 'X-Content-Type-Options nosniff' }
+  end
+end
+
+control 'nginx-12' do
+  impact 1.0
+  title 'TLS Protocols'
+  desc 'When choosing a cipher during an SSLv3 or TLSv1 handshake, normally the client\'s preference is used. If this directive is enabled, the server\'s preference will be used instead.'
+  ref 'SSL Hardening config', url: 'https://mozilla.github.io/server-side-tls/ssl-config-generator/'
+  describe file(nginx_hardening) do
```

Comment:
Going forward we may want to use the nginx_conf of InSpec https://github.com/chef/inspec/blob/master/lib/resources/nginx_conf.rb. Lets keep the current approach, since it works with all InSpec versions